docs/building-on-tigerbeetle/README.md

```diff
@@ -0,0 +1,49 @@
+# Building on TigerBeetle
+
+TigerBeetle is a domain-specific, Online Transaction Processing (OLTP) database. It has a fixed
+schema consisting of [`Account`s](../reference/accounts.md) and
+[`Transfer`s](../reference/transfers.md). In return for this prescriptive design, it provides
+excellent performance, integrated business logic, and powerful invariants.
+
+To help you get started building on TigerBeetle, here is a quick overview of the most important
+concepts:
+
+## Data Model Overview
+
+- [`Account`s](../reference/accounts.md) track balances for users or other entities.
+- [`Transfer`s](../reference/transfers.md) move funds between `Account`s.
+- Account balances and transfer amounts are represented as debits and credits. Double-entry
+  bookkeeping ensures your accounting maintains consistency.
+- Accounts are partitioned into [Ledgers](./data-modeling.md#ledgers), which may represent different
+  currencies, assets, liabilities, etc. or they may be used to support multitenancy. Only accounts
+  on the same ledger can transact directly, but you can use atomically linked transfers to implement
+  [cross-currency transactions](./recipes/currency-exchange.md).
+- TigerBeetle has first-class support for [two-phase transfers](./two-phase-transfers.md), which can
+  hold funds in a pending state and can be used to synchronize transfers with external systems.
+
+## TigerBeetle in Your System Architecture
+
+TigerBeetle is an Online Transaction Processing (OLTP) database built for safety and performance.
+
+It is not a general purpose database, like PostgreSQL or MySQL. Instead, TigerBeetle works alongside
```

Comment:
> It is not a general purpose database, like PostgreSQL or MySQL.

This seems ambiguous... It can be read as meaning "like Postgres and Mysql, TB is not a general purpose database". Maybe remove the comma or reword?